src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/EventSource.cs

```diff
@@ -2879,7 +2878,10 @@ private static bool AttributeTypeNamesMatch(Type attributeType, Type reflectedAt
         // Use reflection to look at the attributes of a class, and generate a manifest for it (as UTF8) and
         // return the UTF8 bytes.  It also sets up the code:EventData structures needed to dispatch events
         // at run time.  'source' is the event source to place the descriptors.  If it is null,
-        // then the descriptors are not creaed, and just the manifest is generated.
+        // then the descriptors are not created, and just the manifest is generated.
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2072:UnrecognizedReflectionPattern",
```

Comment:
This is kind of a large method to be suppressing `IL2072` for the whole thing. Would it be better if we refactored out just the one place that needs to be suppressed in a separate method?